gradle.properties

```diff
@@ -0,0 +1,7 @@
+baseUrl=
+token=
+username=
+password=
+schemasPath=src/test/resources/schemas/
+filterTags=
+cucumberThreadCount=1
```

Comment:
In the specification of the task, it is necesary to define these others properties:
clientId=
clientSecret=
keyToken=

Please, consider add them.